samples/notebooks/python/6-memory-and-embeddings.ipynb

```diff
@@ -433,7 +433,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.10"
+   "version": "3.11.3"
   }
  },
  "nbformat": 4,
```

Comment:
This sample calls code_block._render_function_call_async which is broken due to misaligned arguments

Reply:
fixed, thanks!